### PR TITLE
[FIX] upgrade: small fixes

### PR DESCRIPTION
**Pull request by Donapi**
task-2684744

### CODE REVIEW

**Review by AntoineVDV — APPROVED** [review]
@robodoo delegate=jcs-odoo

File: extensions/odoo_theme/layout_templates/homepage.html [hunk 1]
```diff
@@ -42,13 +42,13 @@ <h5 class="text-muted pt-3 text-uppercase fw_semibold">{{ _("Top Links") }}</h5>
...
-                            {{ _("System configuration") }}
+                        <a href="{{ pathto('administration/maintain/update') }}" class="stretched-link">
+                            {{ _("Bugfix updates") }}
                         </a>
                     </li>
                     <li>
-                        <a href="{{ pathto('administration/maintain/update') }}" class="stretched-link">
-                            {{ _("Bugfix updates") }}
+                        <a href="{{ pathto('administration/upgrade') }}" class="stretched-link">
+                            {{ _("Upgrading Odoo") }}
```
Comment (jcs-odoo) [inline extensions/odoo_theme/layout_templates/homepage.html:51]: Okay why not indeed. I agree that upgrading may be more interesting than deploying ^^

**Review by jcs-odoo — CHANGES_REQUESTED** [review]

File: extensions/odoo_theme/layout_templates/homepage.html (hunk 1 above)
Comment (AntoineVDV) [reply extensions/odoo_theme/layout_templates/homepage.html:51]: The idea is to bring back the "maintain" section on the homepage when it is reworked to have a bit more structure and a general intro.

**Review by jcs-odoo — APPROVED** [review]

**Comment by jcs-odoo** [comment]
@robodoo r+

**Pull request closed without merging**